tests/test_layout_stride.cpp

```diff
@@ -107,7 +107,8 @@ TEST(TestLayoutStrideListInitialization, test_list_initialization) {
   ASSERT_EQ(m.extents().extent(1), 32);
   ASSERT_EQ(m.stride(0), 1);
   ASSERT_EQ(m.stride(1), 128);
-  ASSERT_EQ(m.strides(), (std::array<std::size_t, 2>{1, 128}));
+  ASSERT_EQ(m.strides()[0], 1);
+  ASSERT_EQ(m.strides()[1], 128);
```

Comment:
Why did you change this?  Can we not compare collections?

Reply:
nope it now returns span